container/charliecloud/parpe_base/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:18.04
+
+ADD . /u18
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use COPY instead of ADD for files and folders](https://app.codacy.com/app/dweindl/parPE/pullRequest?prid=3661955)